src/data/buildings.json

```diff
@@ -0,0 +1,21 @@
+[{"id":1,"Address":"19097 Monument Avenue","boilersId":[72,42,89,25,56,48,32],"fullName":"McLaughlin Inc","phone":"379-637-8783"},
```

Comment:
Same observation as for Customer.
biolersId will be later added to Boilers when designing DB schema.
In that case will be: 
Boilers:
- id
- **id_building**
- type
- ...